configure.ac

```
@@ -983,12 +983,12 @@ AC_SUBST([FF_generic_petsc],"$enable_generic")
 ff_petsc_ok=no
 ff_slepc_ok=no
 if test "$with_petsc" != no; then
-    for d in "$with_petsc" "${prefix_petsc}/r/lib" /usr/local/ff++/petsc/lib  /usr /usr/local /opt/usr  /opt/local ;do
+    for d in "$with_petsc" "${prefix_petsc}/r/lib" /usr/local/ff++/petsc/lib  /usr /usr/local /opt/usr  /opt/local /usr/lib/petsc ;do
 	for dd in "" "/petsc/conf/petscvariables" "/lib/petsc/conf/petscvariables" ; do
 	    ffconfpetsc="$d$dd"
 		ffconfslepc="$d/slepc/conf/slepcvariables"
 
-		# echo " petsc ... $ffconfpetsc"
+		echo " petsc ... $ffconfpetsc"
```

Could you please revert that change? I'll merge afterwards.
```suggestion
		# echo " petsc ... $ffconfpetsc"
```

---

Of course! I forgot to comment again the line after fixing the path. Thanks for remark.